src/common/settings.js

```diff
@@ -613,6 +623,13 @@ function createHTMLOptionEx(name, option, short = false) {
                 )
             )
         );
+    } else if (option.type == "info") {
+        e = E.li({ class: "list-group-item beyond20-option beyond20-option-info" },
+            E.label({ class: "list-content", for: name },
+                E.h4({}, title),
+                ...description_p
+            )
+        );
```

Comment:
oh nice, I was going to say, you shouldn't need a setting for this, then realized it's a new info type. The \`\`\` in the description would look weird since this isn't markdown though.